source/data-processing.rst

```diff
@@ -0,0 +1,53 @@
+.. _doc_routinator_data_processing:
+
+Data Processing
+===============
+
+Fetching
+--------
+
+There are two protocols in use to transport RPKI data: rsync and the RPKI
+Repository Delta Protocol (RRDP), which relies on HTTPS. RRDP was designed to be
+the successor to rsync. As almost all RPKI repositories currently support both
+protocols, Routinator will prefer RRDP if available. 
+
+When traversing the RPKI tree, Routinator will find several pointers to child
+publication points, such as the ones operated by National Internet Registries
+and organisations running delegated RPKI. Each pointer explicitly states if RRDP
+is supported in addition to rsync. 
+
+If an RRDP endpoint is unavailable but it has worked in the past, Routinator
+will assume this is a transient problem. It will retry using RRDP for 60 minutes
```

Comment:
It will actually pick a random time between the refresh time and the `rrdp-fallback-time` config value (60 minutes by default) for each repository.

Reply:
I read that but I wasn't sure if it was worth mentioning it. Do you think it should?

Reply:
I think it kinda is worth mentioning, since you may see fallback less than an hour after the initial RRDP failure.

Reply:
So "_up to_ an hour". That's actually quite fancy.